app/epoch/[epoch]/page-client.tsx

```diff
@@ -108,13 +110,13 @@ function EpochOverviewCard({ epoch }: OverviewProps) {
                     <tr>
                         <td className="w-100">First Slot</td>
                         <td className="text-lg-end font-monospace">
-                            <Slot slot={firstSlot} />
+                            <Slot slot={Number(firstSlot)} />
```

Comment:
Can we update `Slot` to take a BigInt instead?

Reply:
We can but it'll blow the changes up a bunch because it's used in lots of places. Might be worth it though, I'll have another look

Reply:
Oh I'm being dumb, just changed Slot to take either. All we do with it is `toString` and `toLocaleString` and that works fine for both!